src/breaking-changes/summary.md

```diff
@@ -4,5 +4,30 @@ Breaking changes should be avoided when possible.
 [RFC 1105](https://rust-lang.github.io/rfcs/1105-api-evolution.html) lays the foundations for what constitutes a breaking change.
 Breakage may be deemed acceptable or not based on its actual impact,
 which can be approximated with a [crater](https://github.com/rust-lang/crater/blob/master/docs/bot-usage.md) run.
+Running crater should be done if nontrivial breakage is expected, so the information is
+available during the final comment period.
 
-If the impact isn't too high, looping in maintainers of broken crates and submitting PRs to fix them can be a valid strategy.
+If the impact isn't too high, looping in maintainers of to-be-broken crates and submitting PRs
+to fix them can be a valid strategy. However, this can only affect the crates in question, and
```

Comment:
```suggestion
to fix them (and ensuring those PRs have been merged and released) can be a valid strategy. However, this can only affect the crates in question, and
```